dashboard/dashboard.html

```diff
@@ -109,17 +109,16 @@
       <figure id="amherst_covid_case_data">
         <figcaption>
           <ul role="list">
-            <li role="listitem"><dfn>Vaccine Phase</dfn><span id="stat_phase"></span><a href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases">Vaccine info</a></li>
-            <li role="listitem"><dfn>COVID Status</dfn><span id="stat_status"></span><a href="https://county.pueblo.org/public-health/covid-19">Code info</a></li>
+            <li role="listitem"><dfn>Vaccine Phase</dfn><span id="stat_phase"></span><a href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases">Vaccine phases info</a></li>
+            <li role="listitem"><dfn>Is Amherst Clinic Open?</dfn><span id="stat_status"></span><a href="https://www.amherstcovid19.org/vaccine">Vaccine appointments</a></li>
           </li>
           </ul>
 
-          <h2>Amherst COVID Data Dashboard</h2>
+          <h2>Amherst COVID-19 Data Dashboard</h2>
           <ul role="list">
-            <li role="listitem"><dfn>Active Cases</dfn><span id="stat_active"></span></li>
-            <li role="listitem"><dfn>Total Cases</dfn><span id="stat_total"></span></li>
-            <li role="listitem"><dfn>2-Week Difference</dfn><span id="stat_delta"></span></li>
-            <li role="listitem"><dfn>Details</dfn><a href="https://docs.google.com/spreadsheets/d/16YiakYsj_PkZjN6PSnY-6xhUDBjEH4hWkcQQPIcxV3w">View Data Table</a></li>
+            <li role="listitem"><dfn>Active Cases in Amherst</dfn><span id="stat_active"></span></li>
```

Comment:
can you see if we can remove the color boxes from these two boxes? It's a left-over css thing that came from moving the info boxes upstairs. It's defined above, you can fix it by making the colors point to an id rather than the first list item.